src/pages/studyView/addChartButton/AddChartButton.tsx

```diff
@@ -303,6 +315,83 @@ class AddChartTabs extends React.Component<IAddChartTabsProps, {}> {
         }
     }
 
+    @computed get existSharedCustomData() {
+        if (this.props.store.isLoggedIn) {
```

Comment:
is this condition correct?  no shared data if user IS logged in?

Reply:
all shared data would be saved to user profile if he is logged in